specification/indices/clear_cache/IndicesClearCacheRequest.ts

```diff
@@ -54,6 +54,12 @@ export interface Request extends RequestBase {
     index?: Indices
   }
   query_parameters: {
+    /**
+     * Comma-separated list of data streams, indices, and aliases used to limit the request.
+     * Supports wildcards (`*`).
+     * To target all data streams and indices, omit this parameter or use `*` or `_all`.
+     */
+    index?: Indices
```

Comment:
The index parameter is read in https://github.com/elastic/elasticsearch/blob/d9e751602dc4a874ef381797844f80044889fbef/server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestClearIndicesCacheAction.java#L43 and has been there since at least 5.0.